.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,8 @@
+Thank you for taking the time to open an issue! If you believe you've found a bug
+please include this section in your issue details. Including a link reproduction
+or failing test will make it easier for us to track down the issue you're having.
```

Comment:
> Thank you for taking the time to open an issue!
>
> For bug reports please include the following section in your issue details. If you include instructions on how to reproduce the bug or a failing test case it will make it easier for us to track down the issue you're having.

opinions?